### PR TITLE
Day3 

### PR DESCRIPTION
**Pull request by dat-nnguyen**
check task 23 please

### CODE REVIEW

File: Day3.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+#task 5,6,7 same as task 4
+
+#task 8 & 9 : Wtf is the slope ? 
+x = int(input())
+y = 2 * x - 2
+z = int(input())
+t = 2 * z - 2
+point_A = x, y 
+point_B = z, t
```
Comment (tuang3142) [inline Day3.py:25]: Do we need `point_A` and `point_B`? They seem redundant. Can we remove them?

File: Day3.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+#task 5,6,7 same as task 4
+
+#task 8 & 9 : Wtf is the slope ? 
+x = int(input())
+y = 2 * x - 2
+z = int(input())
+t = 2 * z - 2
+point_A = x, y 
+point_B = z, t 
+slope = (t - y) / (z - t)   #What if happen division by zero ?
```
Comment (tuang3142) [inline Day3.py:26]: Good question. Why don't you try running this in python and see what happen :dog: 
```
x = 1/0
print(x)
```
Tell me what you find by replying here.

File: Day3.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+#task 4 
+triagle_base = float(input())
+triagle_height = float(input())
+triagle_area = triagle_base * triagle_height * 0.5
+print('The area of the triangle is', triagle_area)
+
+#task 5,6,7 same as task 4
+
+#task 8 & 9 : Wtf is the slope ?
```
Comment (tuang3142) [inline Day3.py:19]: - https://www.khanacademy.org/math/algebra/x2f8bb11595b61c86:linear-equations-graphs/x2f8bb11595b61c86:slope/a/slope-review
- https://happymath.vn/he-so-goc-trong-duong-thang-toan-lop-7-8-toan-quoc-te

File: Day3.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+#task 10 
+#task 11 
+X = int(input())
+Y = X ** 2 + 6 * X + 9
+print(Y)
+
+#task 12
+var_1 = len("python")
+var_2 = len("dragon")
+print(var_1 > var_2)
```
Comment (tuang3142) [inline Day3.py:38]: What do the `print` statement show here? Is this what you expected?

File: Day3.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+ 
+#task 13 
+print("on" in "python" and 'on' in "dragon")
```
Comment (tuang3142) [inline Day3.py:41]: Can you tell me the value of x and y in the code below without wiring the code :dog: 
```
x = 1 in [1, 2, 3, 4]
y = 2 in ['1', '2', '3']
```

**Review by tuang3142 — COMMENTED** [review]
Good job :green_apple: Please address/answer the comments.

File: Day3.py (hunk 5 above)
Comment (dat-nnguyen) [reply Day3.py:41]: True
False cuz 2 is int and '2' is a string

File: Day3.py (hunk 4 above)
Comment (dat-nnguyen) [reply Day3.py:38]: The requirement of the exercise want a False answer

File: Day3.py (hunk 2 above)
Comment (dat-nnguyen) [reply Day3.py:26]: ZeroDivisionError: division by zero

File: Day3.py (hunk 1 above)
Comment (dat-nnguyen) [reply Day3.py:25]: yeh we can

**Review by tuang3142 — APPROVED** [review]
Looking good 🍎

**Pull request merged**